rholang/src/main/scala/coop/rchain/rholang/interpreter/Interpreter.scala

```diff
@@ -34,6 +34,8 @@ trait Interpreter[F[_]] {
 
   def execute(runtime: Runtime[F], reader: Reader): F[Runtime[F]]
 
+  def evaluate(runtime: Runtime[F], term: String, initialPhlo: Cost): F[EvaluateResult]
```

Comment:
This should probably be `evaluateSource` or `evaluateString` since, as you've noted, we have at least three different evaluation sources, i.e., `String`, `Proc`, and `Par`.